cadence/transactions/nftStorefront/cleanup_expired_listings.cdc

```diff
@@ -0,0 +1,27 @@
+import NFTStorefrontV2 from "../contracts/NFTStorefrontV2.cdc"
+
+/// Transaction to facilitate the cleanup of the expired listings of a given
+/// storefront resource account holder.
```

Comment:
Leaving this as is because I see official cadence contracts commenting blocks like `///` (see https://github.com/onflow/nft-storefront/blob/main/transactions/cleanup_expired_listings.cdc)